pages/cart.vue

```diff
@@ -121,12 +123,16 @@ export default defineComponent({
 
     useAsync(async () => {
       await loadCart()
+      const filterOperator = `code eq`
+      const locationCodes = cart.value?.items
+        .filter((item) => {
+          return item.fulfillmentMethod === "Pickup"
+        })
+        .map((item) => item.fulfillmentLocationCode)
+      const filter = locationCodes?.map((code) => `${filterOperator} ${code}`).join(" or ")
```

Comment:
this can be chained together in one pass

` const locationCodeFilter = cart.value?.items
        .filter((item) => {
          return item.fulfillmentMethod === "Pickup"
        })
        .map((item) => ${filterOperator} ${item.fulfillmentLocationCode}).join(" or ")
     `

Reply:
Done